vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+output:
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Mathematical methods for baselinenowcast}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# High-level overview
+
+At a high-level, the method to generate nowcasts use a conceptually simple
```

Comment:
in a section called high level do we need to start high level. 

I would drop both uses and just call this an overview. I would also not say its conceptually simple as for some people it may not feel that way